Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:16.04
+MAINTAINER nop@nop.com
```

Comment:
`MAINTAINER` is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)

Reply:
Done.